docs/components/alert/data.json

```diff
@@ -1,5 +1,6 @@
 {
   "title": "Alert",
+  "mainurl": "/warp-portal-poc/components/alert/",
```

Comment:
I added this, is it already available elsewhere?

Reply:
Yes, it's dynamically set in Component Overview's [index.md's script tag](https://github.com/warp-ds/warp-portal-poc/blob/main/docs/components/index.md?plain=1#L8).

Reply:
You should be able to read the url path under `component.href` in `index.md`

Reply:
Isnt that just the first bit of it though?  Is the specifik path to  /components/alert/ available when it loops through the cards?